components/esm/objectstate.hpp

```diff
@@ -45,7 +45,7 @@ namespace ESM
         {}
 
         /// @note Does not load the CellRef ID, it should already be loaded before calling this method
-        virtual void load (ESMReader &esm);
+        virtual void load (ESMReader &esm, bool inInventory = false);
```

Comment:
Not sure was it intended but this method and load from all derived classes has different declarations now.
```c++
CreatureStats c;
ObjectState* o = &c;
o->load(esm, true); // will not call CreatureStats::load
```
I'd mark all load and save functions in derived from ObjectState classes with `override` or `final` to verify this virtual before applying this pr.